src/services/codefixes/convertToTypeOnlyImport.ts

```diff
@@ -148,3 +155,10 @@ function doChange(changes: textChanges.ChangeTracker, sourceFile: SourceFile, de
         }
     }
 }
+
+function skipAlias(checker: TypeChecker, symbol: Symbol) {
+    while (symbol.flags & SymbolFlags.Alias) {
+        symbol = checker.getAliasedSymbol(symbol);
+    }

```

Comment:
This would have the wrong effect if the type and value locations were swapped in my symbol merging example. What you really want is `getAllSymbolFlags` from checker.ts, which you could export as `@internal`. It also has an `includeTypeOnlyMeanings` parameter, which I think could eliminate the need to call `checker.getTypeOnlyAliasDeclaration`.

Reply:
I considered `symbolIsValue` but opted not to extend the `checker` interface.